src/main.rs

```diff
@@ -70,6 +67,77 @@ impl BFInterp {
             MetadataOperand => todo!(),
         }
     }
+    
+    fn load(&mut self, op: &Operand, dest: &llvm_ir::name::Name) {
```

Comment:
Same as @vext01 noted above. You've imported `Name` so you don't need all the preamble.